Description: fix recent document launch with space in name
 .
 handymenu (4.0-23) unstable; urgency=medium
 .
   * suppression du module des sites visités
Author: arnault perret <arpinux@member.fsf.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- handymenu-4.0.orig/hm_utils.py
+++ handymenu-4.0/hm_utils.py
@@ -130,7 +130,11 @@ def move_app(config, section, app, index
 
 def open_cmd(cmd):
     try:
-        c = [ w for w in cmd.split(' ') ]
+        if cmd.startswith("xdg-open"): 
+            # cas du module récents et des raccourcis fichiers
+            c = [ "xdg-open", "".join(cmd.split(' ', 1)[1:]) ]
+        else:
+            c = [ w for w in cmd.split(' ') ]
         subprocess.Popen(c)
         return True
     except :
@@ -192,7 +196,7 @@ def get_recently_used(max):
             if os.path.isfile(path):
                 name = os.path.basename(path)
                 generic = name 
-                cmd = "xdg-open {}".format(path)
+                cmd = 'xdg-open {}'.format(path)
                 #mimetype = mimetypes.guess_type(path)[0]
                 #if mimetype:
                 #    icon = mimetype.replace('/','-')
